CHANGELOG.md

```diff
@@ -47,7 +47,8 @@ scheme are considered to be bugs.
 ### Miscellaneous
 
 * [#465](https://github.com/intridea/hashie/pull/465): Clean up our RuboCop configuration and fix the outstanding line length violations. This involved some minor refactoring on `Hashie::Extensions::Coercion`, `Hashie::Extensions::Dash::IndifferentAccess`, `Hashie::Extensions::DeepLocate`, `Hashie::Extensions::Mash::SafeAssignment`, and `Hashie::Hash`, but none that were detectable via the test suite - [@michaelherold](https://github.com/michaelherold).
-* [#482](https://github.com/intridea/hashie/pull/482): Update Travis configs to make jruby builds run on trusty dist. - [@BobbyMcWho](https://github.com/BobbyMcWho).
+* [#482](https://github.com/intridea/hashie/pull/482): Update Travis configs to make jruby builds run on trusty dist. - [@bobbymcwho](https://github.com/BobbyMcWho).
+* [#491](https://github.com/intridea/hashie/pull/491): Update rubocop config to max line length of 100. - [@bobbymcwho](https://github.com/BobbyMcWho).
```

Comment:
Remove the trailing period here after 100. Generally we don't need a changelog line for this one.